src/code42cli/cmds/legal_hold.py

```diff
@@ -105,19 +117,32 @@ def show(state, matter_id, include_inactive=False, include_policy=False):
     memberships = _get_legal_hold_memberships_for_matter(
         state.sdk, matter_id, active=active
     )
-    active_usernames = [
-        member["user"]["username"] for member in memberships if member["active"]
-    ]
-    inactive_usernames = [
-        member["user"]["username"] for member in memberships if not member["active"]
-    ]
 
     formatter = OutputFormatter(OutputFormat.TABLE, _MATTER_KEYS_MAP)
     formatter.echo_formatted_list([matter])
-    _print_matter_members(active_usernames, member_type="active")
 
+    users = [
+        [member["active"], member["user"]["userUid"], member["user"]["username"]]
+        for member in memberships
+    ]
+
+    usernames = [user[2] for user in users if user[0] is True]
+    _print_matter_members(usernames, member_type="active")
     if include_inactive:
-        _print_matter_members(inactive_usernames, member_type="inactive")
+        usernames = [user[2] for user in users if user[0] is not True]
+        _print_matter_members(usernames, member_type="inactive")
+
+    if include_devices:
+        user_dataframe = _build_user_dataframe(users)
+        devices_dataframe = _merge_matter_members_with_devices(
+            state.sdk, user_dataframe
+        )
+        if len(devices_dataframe.index) > 0:
+            echo("\nMatter Members and Devices:\n")
+            click.echo(devices_dataframe.to_csv())
```

Comment:
We typically use table format for show commands.

Reply:
List commands, however, are known to support various output formats, inc. CSV.... My first comment on this PR suggests that this would make sense as a `code42 devices list --matter-id <UID>` command, which then you can tack on `-f CSV` or `-f JSON` etc.... Would that be better?